tests/UnimodMapper_test.py

```diff
@@ -160,8 +178,10 @@ def test_conversion(self, conversion):
         #         self.assertEqual(system_exit_check.exception.code, 1)
 
     def test_write(self):
-        xml_file = os.path.join(os.path.dirname(__file__), "test_only_unimod.xml")
-        assert os.path.exists(xml_file) is False
+        xml_file = Path(__file__).parent.joinpath("test_only_unimod.xml")
+        if xml_file.exists():
+            xml_file.unlink()
```

Comment:
unlink() would remove any existing file, correct?
I don't think the tests should remove existing files if a user happens to name it the same way as the test ...

Reply:
You are right